config.go

```diff
@@ -105,8 +106,14 @@ func (cors *cors) validateOrigin(origin string) bool {
 	if cors.allowAllOrigins {
 		return true
 	}
+	r, _ := regexp.Compile("^\\/(.+)\\/[gimuy]?$")
```

Comment:
Move `regexp.Compile` to the top block as a global variable for better performance.